superset/connectors/druid/models.py

```diff
@@ -449,7 +460,7 @@ def get_perm(self) -> Optional[str]:
 
     @classmethod
     def import_obj(cls, i_metric: "DruidMetric") -> "DruidMetric":
-        def lookup_obj(lookup_metric: DruidMetric) -> Optional[DruidMetric]:
+        def lookup_obj(lookup_metric: "DruidMetric") -> Optional["DruidMetric"]:
```

Comment:
Not sure how a nested function referencing its parent method's class is handled by mypy, but if this was working before, we should probably be ok leaving this without quotes?